test/infra/anthos-managed/namespaces/secretmanager-csi-build/integration-test-resources.yaml

```diff
@@ -53,3 +80,22 @@ spec:
     - role: roles/secretmanager.secretAccessor
       members:
         - serviceAccount:k8s-csi-test@secretmanager-csi-build.iam.gserviceaccount.com
+---
+# See
+# https://github.com/kubernetes/test-infra/blob/master/prow/prow_secrets.md#prow-secrets-management
+# and
+# https://github.com/kubernetes/test-infra/blob/master/config/prow/cluster/build_kubernetes-external-secrets_customresource.yaml
+# for information on secret syncing.
+apiVersion: iam.cnrm.cloud.google.com/v1beta1
+kind: IAMPolicy
+metadata:
+  name: k8s-oss-prow-binding
+spec:
+  resourceRef:
+    apiVersion: secretmanager.cnrm.cloud.google.com/v1beta1
+    kind: SecretManagerSecret
+    name: k8s-oss-prow
+  bindings:
+    - role: roles/secretmanager.secretAccessor
+      members:
+        - serviceAccount:kubernetes-external-secrets-sa@k8s-prow.iam.gserviceaccount.com
```

Comment:
I think this should be kubernetes-external-secrets-sa@k8s-prow-builds.iam.gserviceaccount.com